src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/OptionSetInfo.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Collections.Generic;
+using Microsoft.PowerFx.Core.Functions;
+using Microsoft.PowerFx.Core.Localization;
+using Microsoft.PowerFx.Core.Types;
+
+namespace Microsoft.PowerFx.Core.Texl.Builtins
+{
+    internal sealed class OptionSetInfoFunction : BuiltinFunction
+    {
+        public override bool IsSelfContained => true;
+
+        public override bool SupportsParamCoercion => false;
+
+        public OptionSetInfoFunction()
+            : base("OptionSetInfo", TexlStrings.AboutOptionSetInfo, FunctionCategories.Text, DType.String, 0, 1, 1, DType.OptionSetValue)
```

Comment:
Likely this needs to have a CheckTypes, and instead of setting the return type to DType.String, set it based on the backing type of the option set (currently, use `isBooleanValued` to determine that, later update to what's in #1034)

Reply:
actually, without that PR we may not have enough info.

Reply:
I'm fine initially limited this to strings (and failing for non-strings), since that's the biggest thing to unblock PVA.
But we should still fail gracefully for non-strings (not crash).